R/pkg/R/utils.R

```diff
@@ -354,8 +354,8 @@ varargsToStrEnv <- function(...) {
       } else {
         value <- pairs[[name]]
         if (!(is.logical(value) || is.numeric(value) || is.character(value) || is.null(value))) {
-          stop(paste0("Unsupported type for ", name, " : ", class(value),
-               ". Supported types are logical, numeric, character and NULL."), call. = FALSE)
+          stop("Unsupported type for ", name, " : ", toString(class(value)), ". ",
```

Comment:
Quick question, why do we need to do `toString`? Were there any differences at `stop(paste(..., ...))` vs `stop(..., ...)`?

Reply:
`stop` uses `sep=''`, so `stop(paste0(...))` is equivalent to `stop(...)`

`toString` collapses in cases `length>1`, compare:

```
stop(toString(class(Sys.time())))
stop(class(Sys.time()))
```